src/library/scala/runtime/NonLocalReturnControl.scala

```diff
@@ -15,7 +15,16 @@ package scala.runtime
 import scala.util.control.ControlThrowable
 
 // remove Unit specialization when binary compatibility permits
-@annotation.nowarn("cat=lint-unit-specialization")
+@annotation.nowarn
 class NonLocalReturnControl[@specialized(Byte, Short, Int, Long, Char, Float, Double, Boolean, Unit) T](val key: AnyRef, val value: T) extends ControlThrowable {
   final override def fillInStackTrace(): Throwable = this
+
+  // Until next re-STARR, suppressing the unit specialization warning (so we can `-Werror) is a bit
+  // tricky.  When compiling locally, we're using STARR, so there's no warning.  On CI, we bootstrap,
+  // and then the warning appears. The only way I can think of to have no warnings either way is to
+  // have the blanket @nowarn, but then also intentionally put something warnable in. So that's
+  // what this is doing here.  After next re-STARR, we can remove this, and then also change the
+  // @nowarn above to be more specific: @annotation.nowarn("cat=lint-unit-specialization")
+  0
```

Comment:
That's pretty baroque. Sorry, I see this is a draft beer. Is it easier and more blankety to suppress all "suppresses nothing" warnings in build.sbt, then remove that after restarr?  Does anyone else miss season one of american idol? Ryan Starr is the same age as Kelly Clarkson.